File: speakers/miliox/samples/basic_class/Person.cpp
// Person.cpp
#include <string>

class Person {
private:
    std::string m_firstName;
    std::string m_lastName;
    int m_age;
public:
    Person(std::string firstName,
           std::string lastName,
           int age) :
        m_firstName(firstName),
        m_lastName(lastName),
        m_age(age) {
    }
};

File: speakers/miliox/samples/basic_class/Person.java
public class Person {
    private String firstName;
    private String lastName;
    private int age;

    public Person(String firstName, String lastName, int age) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.age = age;
    }

    private void print() {
        System.out.printf("%s %s %d\n",
            this.firstName, this.lastName, this.age);
    }
}


